parliament.gemspec

```diff
@@ -34,6 +34,7 @@ Gem::Specification.new do |spec|
   spec.add_development_dependency 'rspec', '~> 3.0'
   spec.add_development_dependency 'rubocop'
```

Comment:
We should include versions for all gems we are dependent on, both for reliability and security reasons